logics/impls/controller/mod.rs

```diff
@@ -267,6 +266,7 @@ pub trait Internal {
         &self,
         account: AccountId,
         pool_attributes: Option<PoolAttributes>,
+        get_asset_param: bool,
```

Comment:
Please tell us why you want to include this flag.

I don't think this flag is necessary.
What this flag does is only push to Vec or not, and the amount of processing (especially by cross contract call) is almost the same, and I think it is confusing because Vec<HypotheticalAccount LiquidityCalculationParam> exists (empty or not) in the return value, so I think the flag makes it more difficult.

Reply:
I thought that reducing vector operation can reduce gas.
But it the gas cost is similar this flag doesn't needed.
I will remove it.

Reply:
I think it will certainly decrease slightly.
However, I suspect that many of them are cross contract calls, and if we have this one, we would also want a flag for AccountCollateralData, which would also complicate things.

Reply:
Yes, I agree,
I have removed the flag.